src/test/java/pages/MainPage.java

```diff
@@ -1,74 +1,186 @@
 package pages;
 
-import elements.Button;
-import elements.Text;
+import elements.*;
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
 import org.openqa.selenium.TimeoutException;
 import org.openqa.selenium.support.FindBy;
-import org.openqa.selenium.support.ui.ExpectedConditions;
 import org.testng.Assert;
-
+import properties.ConfigStorage;
+import util.WaitUtils;
 
 public class MainPage extends BasePage {
 
+    private final String MAIN_URL = ConfigStorage.getMainUrl();
+
     @FindBy(className = "auth-bar__item--text")
-    public Button entranceButton;
+    private Button entranceButton;
 
     @FindBy(className = "auth-bar__item--cart")
-    public Button shoppingCart;
+    private Button shoppingCartButton;
+
+    @FindBy(className = "auth-form__label-title")
+    private Text loginVia;
```

Comment:
add element type to the name, use the same pattern for names for all elements

Reply:
added